src/Message/ApplePayAuthorizeRequest.php

```diff
@@ -0,0 +1,362 @@
+<?php
+
+namespace Omnipay\Vindicia\Message;
+
+use Omnipay\Common\Message\ResponseInterface;
+
+/**
+ * Retrieve an Apple Pay Payment Session object.
+ *
+ * Post a request to Apple Pay server's Payment Session endpoint using two-way TLS.
+ *
+ * Parameters:
+ * - pemCertPath: Path to the cert of the split Merchant Identity certification,
+ * associated with your merchantID needed to make the call.
+ * - keyCertPath: Path to the key cert of the split Merchant Identity certification,
```

Comment:
these are gateway parameters, not request parameters, right? they don't need to be documented with the request if so

Reply:
They are gateway parameters but I thought they should be documented here because we're using them in the request object. Also, it would be less confusing as its in the example object that Apple Requests in the API docs: https://developer.apple.com/documentation/apple_pay_on_the_web/apple_pay_js_api/requesting_an_apple_pay_payment_session

Reply:
OK, can you specify that they get set by the gateway then?